mbed_greentea/mbed_test_api.py

```diff
@@ -240,124 +132,57 @@ def get_auto_property_value(property_name, line):
     # Select who will digest test case serial port data
     if digest_source == 'stdin':
         # When we want to scan stdin for test results
-        obs = StdInObserver()
+        raise NotImplementedError
     elif digest_source is not None:
         # When we want to open file to scan for test results
-        obs = FileObserver(digest_source)
-    else:
-        # Command executing CLI for host test supervisor (in detect-mode)
-        cmd = ["mbedhtrun",
-                '-d', disk,
-                '-p', port,
-                '-f', '"%s"'% image_path,
-                ]
+        raise NotImplementedError
+
+    # Command executing CLI for host test supervisor (in detect-mode)
+    cmd = ["mbedhtrun",
+            '-d', disk,
+            '-p', port,
+            '-f', '"%s"'% image_path,
+            ]
+
+    # Add extra parameters to host_test
+    if program_cycle_s is not None:
+        cmd += ["-C", str(program_cycle_s)]
+    if copy_method is not None:
+        cmd += ["-c", copy_method]
+    if micro is not None:
+        cmd += ["-m", micro]
+    if reset is not None:
+        cmd += ["-r", reset]
+    if reset_tout is not None:
+        cmd += ["-R", str(reset_tout)]
```

Comment:
Is reset_tout = reset_timeout ? I would guess as it would be integer thus casting to str(). Then above ``` program_cycle_s ``` is string or _s as seconds?